src/fixtures/telemetry_collectors/working_collector.ts

```diff
@@ -85,25 +85,25 @@ export const myCollector = makeUsageCollector<Usage>({
     },
     my_objects: {
       total: {
-        type: 'number',
+        type: 'long',
       },
       type: { type: 'boolean' },
     },
     my_array: {
       type: 'array',
       items: {
         total: {
-          type: 'number',
+          type: 'long',
         },
         type: { type: 'boolean' },
       },
     },
     my_str_array: { type: 'array', items: { type: 'keyword' } },
     my_index_signature_prop: {
-      count: { type: 'number' },
-      avg: { type: 'number' },
-      max: { type: 'number' },
-      min: { type: 'number' },
+      count: { type: 'long' },
+      avg: { type: 'long' },
```

Comment:
Strictly speaking, this should be a float.
```suggestion
      avg: { type: 'float' },
```